components/camera/videosource/webcam.go

```diff
@@ -565,6 +565,10 @@ func (c *monitoredWebcam) Stream(ctx context.Context, errHandlers ...gostream.Er
 	return c.exposedSwapper.Stream(ctx, errHandlers...)
 }
 
+func (c *monitoredWebcam) GetImage(ctx context.Context) (image.Image, func(), error) {
+	return camera.ReadImage(ctx, c.underlyingSource)
+}
+
```

Comment:
Stretch goal - okay to do in another pr -  just for webcam, can we rely on the return from media devices directly rather than what we've bundled in the underlying source?

Reply:
https://viam.atlassian.net/browse/RSDK-9204?atlOrigin=eyJpIjoiZjk0MjY0Mjk4OWZkNGY2YWJhNDIzMTUzMjk5NjY0MGUiLCJwIjoiaiJ9